resources/views/header-script.blade.php

```diff
@@ -16,7 +16,10 @@
             window.dataLayer.push({!! $item->toJson() !!});
         @endforeach
 
-        window.addEventListener('push_to_data_layer', (event) => window.dataLayer.push(event.detail))
```

Comment:
was this call specific to Elevar?  Since we are in GTM I want to ensure no one else is using it.